content/reference-transaction-format.md

```diff
@@ -577,6 +583,8 @@ These codes indicate an error in the local server processing the transaction; it
 
 These codes indicate that the transaction was malformed, and cannot succeed according to the XRP Ledger protocol. They have numerical values in the range -299 to -200. The exact code for any given error is subject to change, so don't rely on it.
 
+**Tip:** Transactions are not applied to ledgers with `tem` codes. A `tem` result is final unless the rules for a valid transaction change. (For example, using functionality from an [Amendment](concept-amendments.html) before that amendment is enabled results in `temDISABLED`; such a transaction could succeed later if it becomes valid when the amendment is enabled.)
```

Comment:
Are these transactions added to the ledger, or, like transactions with `tef` codes, are they left off? If so, I suggest:
> **Tip:** Transactions with `tem` codes are not applied to ledgers.

Reply:
Yep, that's a misplaced modifier. Good catch. Revised to be more like the others:

> **Tip:** Transactions with `tem` codes are not applied to ledgers, and cannot cause any changes to XRP Ledger state.